php/libraries/LorisForm.class.inc

```diff
@@ -373,7 +373,7 @@ class LorisForm
                 $optionKey = (string) $optionKey;
             }
 
-            if ($optionKey === $this->getValue($el['name'])) {
+            if ($optionKey === $el['selected']) {
```

Comment:
I think the getValue (the user submitted data) needs to take precedence over "$el['selected']" (the form author specified a default), so the if/elseif should be the other order.